src/wire/libevent_callbacks.cpp

```diff
@@ -149,8 +173,13 @@ void StateMachine(LibeventSocket *conn) {
 
         if (conn->pkt_manager.is_started == false) {
           // We need to handle startup packet first
-          status = conn->pkt_manager.ProcessStartupPacket(&conn->rpkt);
-          conn->pkt_manager.is_started = true;
+          int status_res = conn->pkt_manager.ProcessInitialPacket(&conn->rpkt);
+          status = (status_res != 0);
+          if (status_res == 1)
```

Comment:
Hi, I have double checked line 153-180 in that file. I see nothing miss its "}". So could you clarify which line misses the {}?